src/Authentication/Authentication/custom/Find-MgGraphPermission.ps1

```diff
@@ -0,0 +1,226 @@
+# ------------------------------------------------------------------------------
+#  Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the MIT License.  See License in the project root for license information.
+# ------------------------------------------------------------------------------
+
+Set-StrictMode -Version 6.0
+
+. "$psscriptroot/common/Permissions.ps1"
+
+# Register the output type of Find-MgGraphPermission so that
+# auto-completion works for commands such as Select-Object
+'Id', 'Consent', 'Name', 'Description', 'PermissionType' | foreach {
+    $typeParameters = @{
+        TypeName = 'Microsoft.Graph.Custom.Permission'
+        MemberType = 'NoteProperty'
+        MemberName = $_
+        Value = $null
+    }
+
+    Update-typedata @typeParameters -force
+}
+
+<#
+.SYNOPSIS
```

Comment:
@msewaweru this is the documentation section -- I believe this will be used to generate the online docs...